airflow-core/src/airflow/ui/src/components/DisplayMarkdownButton.tsx

```diff
@@ -20,9 +20,12 @@ import { Box, Heading, VStack } from "@chakra-ui/react";
 import { type ReactElement, useState } from "react";
 
 import { Button, Dialog } from "src/components/ui";
+import { ResizableWrapper } from "src/components/ui/ResizableWrapper";
 
 import ReactMarkdown from "./ReactMarkdown";
 
+const STORAGE_KEY = "airflow-markdown-dialog-size";
```

Comment:
All dag documentation + ti documentation will share the same setting / key. I'm fine with that, I just wanted to confirm